flask_restplus/api.py

```diff
@@ -621,6 +622,9 @@ def handle_error(self, e):
         for header in HEADERS_BLACKLIST:
             headers.pop(header, None)
 
+        if not current_app.config.get("ERROR_INCLUDE_MESSAGE", True):
```

Comment:
What if you don't want to include a message field by default, but you still want to return message field sometimes. In this way, you can never return a `message` field.

Reply:
It is all or nothing at the moment. Do we really want to handle this scenario? In the end, if you still want to include a message in some cases it is something you can add in the @api.errorhandler method.